emsurfer/src/app/components/header/header.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="container-fluid">
```

Comment:
[red]
Make sure that in VS Code you have: TSLint 1.0.0 by Microsoft, Prettier - Code Formatter 1.8.1 by Esben Petersen and Formatting Toggle 1.4.0 by tombonnike (and make sure formatting is enabled)

Reply:
Done